### DISCUSSION TITLE
查询oracle表的时候 用IPage分页 动态拼接的还是mysql的LIMIT关键字 怎么解决？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#270] [02.提问]
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/6092525/230600578-745d5302-e02a-4a58-b9b6-ab9d4f3f95d0.png">
<img width="480" alt="image" src="https://user-images.githubusercontent.com/6092525/230600684-07a8ca9f-9306-4efb-954a-5fbc6a7265c2.png">
<img width="272" alt="image" src="https://user-images.githubusercontent.com/6092525/230600630-ebcafab5-0764-426b-b6ab-7bb601b9716c.png">
如何让mybatis用oracle的分页逻辑？

**Comment by zuihou** [comment]
mybatis-plus分页插件要指定数据库类型，看它官网

**Reply by zqb666** [reply]
<img width="851" alt="image" src="https://user-images.githubusercontent.com/6092525/230601487-bf0e569d-b3f4-4941-99c9-730a841d02ca.png">
加了这个配置没用

**Reply by zqb666** [reply]
<img width="837" alt="image" src="https://user-images.githubusercontent.com/6092525/230602801-0986422d-a1ee-4b05-ba2e-9d909c10772e.png">
这样可以 但是这样整个服务只能用oracle服务了 请教下有没有办法支持多种数据库分页的配置？

**Reply by zuihou** [reply]
这个你得去问mybatis-plus
